### PR TITLE
Update dashHtmlComponents package in R to 0.14.0

### PR DESCRIPTION
**Pull request by rpkyle**
In addition to updating `dash_html_components.min.js` and `dash_html_components.dev.js` to version 0.14.0, this PR also aims to incorporate support for `loading_state` in "transpiled" dashR components.

Some simplification of the component generation statements in `package.json` is proposed as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     "lint": "eslint src scripts",
     "build:js": "webpack --mode production",
     "build:js-dev": "webpack --mode development",
-    "build:py": "node ./extract-meta src/components > dash_html_components/metadata.json && cp package.json dash_html_components && dash-generate-components ./src/components dash_html_components",
-    "build:r": "node ./extract-meta src/components > dash_html_components/metadata.json && cp package.json dash_html_components && dash-generate-components ./src/components dash_html_components --r-prefix 'html'",
+    "build:py": "dash-generate-components ./src/components dash_html_components",    
+    "build:r": "dash-generate-components ./src/components dash_html_components --r-prefix 'html'",
```
Comment (alexcjohnson) [inline package.json]: Ha good call, `dash-generate-components` does all of this and has for quite some time!

Comment (alexcjohnson) [reply package.json]: In fact now we can 🔪 `extract-meta` from this repo, right?

Comment (rpkyle) [reply package.json]: :beers:

Comment (rpkyle) [reply package.json]: fixed in f3f33856d95be679db1ceb931502b0ea819712c4

**Pull request merged**